src/components/teasers/teaser.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+  TeaserTitle,
+  TeaserTitleLarge,
+  Text,
+  Timestamp,
+  Topline,
+} from '../typography/typography';
+import { Arrow } from '../icons/arrow';
+import { theme } from '../layout/theme';
+import { Link } from '../links/links';
+
+const TeaserContainer = styled.div`
+  cursor: pointer;
+  overflow: hidden;
+
+  img {
+    transition: transform 0.2s;
+  }
+
+  &:hover {
+    color: ${theme.palette.text.topline};
+
+    p {
+      color: ${theme.palette.text.topline};
+    }
+
+    img {
+      transform: scale(1.05);
+    }
```

Comment:
Is this effect agreed? Feels some awkward that the image zooms in this case. But let's keep it and I would like to come back to this.

Reply:
> If it contains an image the image is zoomed in on hover and back out on mouseOut
That's the description in the design. Maybe we need to adjust the values (time and scale factor)